demo/radar.html

```diff
@@ -606,13 +626,16 @@
 		}
 
 
-// TODO Just for testing:
-// tweenInitAll();
+		// TODO Just for testing:
```

Comment:
Are these two lines necessary for testing each time?

Reply:
Removed